ExternalTasks/NodeJs/main.js

```diff
@@ -1,70 +1,68 @@
 const {HttpClient} = require('@essential-projects/http');
+
 const {
   ExternalTaskApiClientService,
   ExternalTaskApiExternalAccessor,
   ExternalTaskWorker,
 } = require('@process-engine/external_task_api_client');
+const {ExternalTaskFinished} = require('@process-engine/external_task_api_contracts');
 
-const {
-  ExternalTaskFinished,
-} = require('@process-engine/external_task_api_contracts');
-
-const identity = {
-    token: 'ZHVtbXlfdG9rZW4=',
+const sampleIdentity = {
+  token: 'ZHVtbXlfdG9rZW4=',
 };
 
-const TOPIC = 'Etikett-ausdrucken';
-const MAX_TASKS = 10;
-const POLLING_TIMEOUT = 1000;
-const WAIT_TIMEOUT = 10000;
+async function main() {
+  const externalTaskWorker = createExternalTaskWorker('http://localhost:8000');
+
+  const topic = 'Etikett-ausdrucken';
+  const maxTasks = 10;
+  const pollingTimeout = 1000;
 
-function createExternalTaskWorker(url) {
-    const httpClient = new HttpClient();
-    httpClient.config = {url: url};
-    
-    const externalAccessor = new ExternalTaskApiExternalAccessor(httpClient);
+  console.log(`Warten auf Aufgaben für das Topic '${topic}'.`);
 
-    const externalTaskAPIService = new ExternalTaskApiClientService(externalAccessor);
+  externalTaskWorker.waitForAndHandle(sampleIdentity, topic, maxTasks, pollingTimeout, async (externalTask) => {
+    console.log('Daten external-Task: ');
+    console.log(externalTask);
+    console.log('');
 
-    const externalTaskWorker = new ExternalTaskWorker(externalTaskAPIService);
+    let result = await doSomeLongWork();
 
-    return externalTaskWorker;
-}
+    let externalTaskFinished = new ExternalTaskFinished(externalTask.id, result);
 
-const doSomeLongWork = async (externalTask) => {
+    return externalTaskFinished;
+  });
+}
 
-    console.log(`Warte für ${WAIT_TIMEOUT} Millisekunden.`);
-    await sleep(WAIT_TIMEOUT);
+function createExternalTaskWorker(url) {
+  const httpClient = new HttpClient();
+  httpClient.config = {url: url};
 
-    const result = { 
-        TestProperty: 'Dies ist das Ergebnis vom JavaScript-external-Task.'
-    };
+  const externalAccessor = new ExternalTaskApiExternalAccessor(httpClient);
 
-    console.log('Bearbeitung fertig!');
+  const externalTaskAPIService = new ExternalTaskApiClientService(externalAccessor);
 
-    return result;
-};
+  const externalTaskWorker = new ExternalTaskWorker(externalTaskAPIService);
 
-async function main() {
-    const externalTaskWorker = createExternalTaskWorker('http://localhost:8000');
+  return externalTaskWorker;
+}
 
-    console.log(`Warten auf Aufgaben für das Topic '${TOPIC}'.`);
+const doSomeLongWork = async (externalTask) => {
```

Comment:
Gibt es einen Grund, dass wir `doSomeLongWork ` als `const` anstatt per `function` deklarieren?
```suggestion
async function doSomeLongWork(externalTask) {
```

Reply:
Hier wird das ganz gut erklärt:
https://softwareengineering.stackexchange.com/questions/364086/why-use-const-foo-instead-of-function-foo

Reply:
Wobei ich grad auch sehe, dass das eine Top-Level function ist. 
Kann man also auch ändern.